modin/pandas/test/test_series.py

```diff
@@ -4540,6 +4540,16 @@ def test_encode(data, encoding_type):
         df_equals(modin_result, pandas_result)
 
 
+@pytest.mark.parametrize("data", test_string_data_values, ids=test_string_data_keys)
+def test_concat_string_to_series(data):
+    modin_series, pandas_series = create_test_series(data)
+
+    modin_series = "random_string" + modin_series
+    pandas_series = "random_string" + pandas_series
+
+    df_equals(modin_series, pandas_series)
+
+
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda s: 'string' + s)
```

Reply:
I keep forgetting about `eval_general`. I'll need to keep this in mind for future tests. I usually go with the habit of following the style in an existing test file.